elfi/methods/bo/utils.py

```diff
@@ -111,28 +111,28 @@ def minimize(fun,
     return locs[ind_min], vals[ind_min]
 
 
-class CostFunction:
-    """Convenience class for modelling acquisition costs."""
+class AdjustmentFunction:
```

Comment:
Why have `AdjustmentFunction`class and then define the maker functions later on that return classes defined with the scope of the maker functions?

Reply:
the maker functions were needed to make new acquisition classes that inherit from the base acquisition class that is given as input. the functions currently take as input the base acquisition class and an `AdjustmentFunction` instance, but i could also remove this convenience class and have `make_additive_acq` and `make_multiplicative_acq` take as input the callable that we want to use as additive or multiplicative term and the callable that returns its gradient. is that what you meant?